articles/application-insights/app-insights-java-get-started.md

```diff
@@ -173,6 +173,61 @@ You can also [set it in code](app-insights-api-custom-events-metrics.md#ikey):
 ## 4. Add an HTTP filter
 The last configuration step allows the HTTP request component to log each web request. (Not required if you just want the bare API.)
 
+### Spring Boot Applications
+Register the Application Insights `WebRequestTrackingFilter` in your Configuration class:
+
+```Java
+package devCamp.WebApp.configurations;
+
+    import javax.servlet.Filter;
+
+    import org.springframework.boot.context.embedded.FilterRegistrationBean;
+    import org.springframework.context.annotation.Bean;
+    import org.springframework.core.Ordered;
+    import org.springframework.beans.factory.annotation.Value;
+    import org.springframework.context.annotation.Configuration;
+    import com.microsoft.applicationinsights.TelemetryConfiguration;
+    import com.microsoft.applicationinsights.web.internal.WebRequestTrackingFilter;
+
+
+    @Configuration
+    public class AppInsightsConfig {
+
+	//Initialize AI TelemetryConfiguration via Spring Beans
+        @Bean
+        public String telemetryConfig() {
+            String telemetryKey = System.getenv("APPLICATION_INSIGHTS_IKEY");
+            if (telemetryKey != null) {
+                TelemetryConfiguration.getActive().setInstrumentationKey(telemetryKey);
+            }
+            return telemetryKey;
+        }
+	
+	//Set AI Web Request Tracking Filter
+        @Bean
+        public FilterRegistrationBean aiFilterRegistration(@Value("${spring.application.name:application}") String applicationName) {
+	       FilterRegistrationBean registration = new FilterRegistrationBean();
+	       registration.setFilter(new WebRequestTrackingFilter(applicationName));
+	       registration.setName("webRequestTrackingFilter");
+	       registration.addUrlPatterns("/*");
+	       registration.setOrder(Ordered.HIGHEST_PRECEDENCE + 10);
+	       return registration;
+       } 
+
+	//Set up AI Web Request Tracking Filter
+        @Bean(name = "WebRequestTrackingFilter")
+        public Filter webRequestTrackingFilter(@Value("${spring.application.name:application}") String applicationName) {
+            return new WebRequestTrackingFilter(applicationName);
+        }	
+    }
+```
+
+This class will configure the `WebRequestTrackingFilter` to be the first filter on the http filter chain. It will also pull the instrumentation key from the operating system environment variable if it is available.
+
+> We are using the web http filter configuration rather than the Spring MFC configuration because this is a Spring Boot application, and it has its own Spring MVC configuration. See the sections below for Spring MVC specific configuration.
```

Comment:
@grlima PR looks fine except one minor typo here. Please replace "Spring MFC" -> "Spring MVC" in the line 227.